Removed namespaced_types

ariel-anieli opened the pull request:
* introduced for handling deprecated types existing before R17
* CI/CD handles R24 up to R26.

ariel-anieli committed:
Removed namespaced_types

* introduced for handling OTP versions that did not support maps
* introduced in af74c264ead4c5dd01037e5ebef5a72c8d0ef6e3
* CI/CD handles R24 up to R26
* R17 and onwards have maps.

Link: https://www.erlang.org/docs/23/man/maps
Signed-off-by: Ariel Otilibili <otilibil@eurecom.fr>

ariel-anieli commented:
Hello @soranoba, any news on the PR?

soranoba commented:
@ariel-anieli 
Please explain why this PR is necessary.
I avoid breaking changes unless there is a good reason.

ariel-anieli commented:
Sure, @soranoba: this message is also included in the commit:
* introduced for handling OTP versions that did not support maps
* introduced in af74c264ead4c5dd01037e5ebef5a72c8d0ef6e3
* CI/CD handles R24 up to R26
* [R17 and onwards have maps](https://www.erlang.org/docs/23/man/maps).

soranoba commented:
That's not the reason.

I don't intend to support old versions, but the motivation to make breaking changes is low, Because major version bump required.

ariel-anieli commented:
I'm closing the PR, no worries, @soranoba:
From your answer, I see you're reluctant to make this change.
